frontend/src/toolbar/shared/ElementMetadata.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { dockLogic } from '~/toolbar/dockLogic'
+import { ActionStep } from '~/toolbar/shared/ActionStep'
+import { CloseOutlined, CalendarOutlined, AimOutlined } from '@ant-design/icons'
+import { heatmapLogic } from '~/toolbar/shared/heatmapLogic'
+import { Button, Statistic, Row, Col, Divider } from 'antd'
+
+export function ElementMetadata({ rect, meta: { actionStep, element: metaElement }, pointerEvents, onClose }) {
+    const { domZoom, domPadding } = useValues(dockLogic)
+    const { elementMap, eventCount } = useValues(heatmapLogic)
+
+    const heatmapMeta = elementMap.get(metaElement)
+
+    return (
+        <>
+            <div
```

Comment:
The X here doesn't look correctly aligned.
![image](https://user-images.githubusercontent.com/1727427/84646503-7d960300-af02-11ea-9cf9-9075442900d6.png)

It's very hard to select elements at the bottom of the page, as the page will start jumping around a lot because this is higher than the page itself.

Reply:
fixed now